### PR TITLE
Added contributor list

### PR DESCRIPTION
**Pull request by pioardi**
A big thanks to my main contributors

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -140,6 +142,19 @@ See guidelines [CONTRIBUTING](CONTRIBUTING.md) <br>
...
+<h2 id="contributors">Contributors</h2>
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+<!-- prettier-ignore-start -->
+<!-- markdownlint-disable -->
+
+**Creator/Owner:**
+* [__Alessandro Pio Ardizio__](https://github.com/pioardi)
+
+***Contributors***
+* [__shinigami92__](https://github.com/Shinigami92)
```
Comment (Shinigami92) [inline README.MD:154]:
```suggestion
* [__Shinigami92__](https://github.com/Shinigami92)
```

**Review by Shinigami92 — CHANGES_REQUESTED** [review]
You could also add the [contributors](https://docs.npmjs.com/cli/v6/configuring-npm/package-json#people-fields-author-contributors) section to `package.json`

**Comment by Shinigami92** [comment]
https://github.com/prettier/plugin-pug/blob/213fafc0db9b3654fab13116cce0bbfe2f3d7a3a/package.json#L20-L22

**Review by Shinigami92 — APPROVED** [review]
It's good from my side, but we need to wait and fill jerome's data instead of `"N/A"`

**Comment by pioardi** [comment]
@jerome-benoit  I will hold on so that if you would like to add more details about you we can do that .)

File: package.json [hunk 2]
```diff
@@ -52,6 +52,18 @@
...
   "author": "pioardi",
+  "contributors": [
+    {
+      "name": "Christopher Quadflieg",
+      "email": "chrissi92@hotmail.de",
+      "url": "https://github.com/Shinigami92"
+    },
+    {
+      "name": "Jérôme Benoit",
+      "email": "jerome.benoit@sap.com",
```
Comment (Shinigami92) [inline package.json:63]:
```suggestion
      "email": "jerome.benoit@piment-noir.org",
```

**Review by Shinigami92 — APPROVED** [review]

**Comment by Shinigami92** [comment]
@pioardi Just a hint: a squash-merge had produced a better commit history

**Pull request merged**